docs/TektonHub.md

```diff
@@ -0,0 +1,96 @@
+# Tekton Hub
+
+TektonHub custom resource allows user to install and manage [Tekton Hub][hub].
+
+TektonHub is an optional component and currently cannot be installed through TektonConfig. It has to be installed seperately. It is available for both Kubernetes and OpenShift platform.
+
+To install Tekton Hub on your cluster follow steps as given below:
+
+1.  Make sure that `tekton-pipelines` namespace is present on your
```

Comment:
we should follow `targetNamespace` similar to other component
by default in config dir CR examples we can have the defaults per platform 
but user should be allowed to install any namespace
in hub case it is straight forward .. instead injecting `tekton-pipelines` ns in resources
we inject spec.targetNamespace